internal/querycoordv2/task/scheduler.go

```diff
@@ -782,6 +781,13 @@ func (scheduler *taskScheduler) checkStale(task Task) bool {
 			log.Warn("the task is stale, the target node is offline")
 			return true
 		}
+		if action.Type() == ActionTypeGrow {
+			isStoppingNode, _ := scheduler.nodeMgr.IsStoppingNode(action.Node())
+			if isStoppingNode {
+				log.Warn("task stale due to dst node for growing has been stopping", zap.Int64("dstNode", action.Node()))
```

Comment:
the "nodeID" has been added into the log at line 777

Reply:
NIT comment
Please fix it later